src/test/java/no/unit/nva/database/intefaces/DynamoEntryTest.java

```diff
@@ -0,0 +1,31 @@
+package no.unit.nva.database.intefaces;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+import no.unit.nva.database.RoleDb;
+import no.unit.nva.database.exceptions.InvalidRoleException;
+import org.junit.jupiter.api.Test;
+
+class DynamoEntryTest {
+
+    public static final String SOME_TYPE = "SomeType";
+
+    @Test
+    void setTypeHasNoEffect() throws InvalidRoleException {
+        RoleDb roleDb = RoleDb.newBuilder().withName("SomeName").build();
```

Comment:
We cannot test DynamoEntry directly on these methods because it has no fields to test. Any anonymous implementation serves no purpose as there is no way to verify that a method does nothing;